docs/app/pages/components/components-sections/notifications/badge/snippets/app.html

```diff
@@ -0,0 +1,30 @@
+<div class="row">
+    <div class="col-sm-3 col-xs-6">
```

Comment:
This file needs to be kept in sync with the documentation example. I usually just copy and paste before committing.